cmd/subnetcmd/join.go

```diff
@@ -197,9 +242,12 @@ but until the node is whitelisted, it will not be able to validate this subnet.`
 	// or, pluginDir was set but not avagoConfigPath
 	// if **both** flags were set, this will be skipped...
 	if avagoConfigPath == "" {
-		avagoConfigPath, err = app.Prompt.CaptureString("Path to your existing config file (or where it will be generated)")
-		if err != nil {
-			return err
+		avagoConfigPath = findAvagoConfigPath()
+		if avagoConfigPath == "" {
```

Comment:
Should we provide an option to override or confirm the found path? Or maybe that would be to just use flags?

Reply:
Hmm. 

I think you're right. We shouldn't just take whatever it found. We should probably give the user a chance to confirm.

Reply:
Updated with a fix